vendor/vendor.json

```diff
@@ -493,6 +493,12 @@
 			"revision": "53a58da551e961b3710bbbdfabbc162c3f5f30f6",
 			"revisionTime": "2018-01-31T23:13:00Z"
 		},
+		{
+			"checksumSHA1": "H1rrbVmeE1z2TnkF7tSrfh+qUOY=",
```

Comment:
Should we let docker vendor pkg version be same ? :)

Reply:
@HusterWan Yes. It's already "53a58da551e961b3710bbbdfabbc162c3f5f30f6"